<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fractal_Dialog_Window</class>
 <widget class="QDialog" name="Fractal_Dialog_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Множина Жюліа</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Icons/Icons/2.png</normaloff>:/Icons/Icons/2.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="5" column="7">
       <widget class="QPushButton" name="IterationForwardButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="ColorScheme">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color Scheme:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="4">
       <widget class="QSpinBox" name="iterationSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QComboBox" name="scaleComboBox">
        <item>
         <property name="text">
          <string>px</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="I_Label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>i</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="Iteration">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Iteration:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="IterationBackwardButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="Constant_C">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Constant C:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QDoubleSpinBox" name="imaginarySpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="8">
       <widget class="QSlider" name="scaleSlider">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="singleStep">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="saveAsButton">
        <property name="text">
         <string>Save As</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Scale">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Scale:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="realSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QSpinBox" name="scaleSpinBox">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="singleStep">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="Plus">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="5" alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="colorComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="HideButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
